### PR TITLE
Brokenlink

### PR DESCRIPTION
**Pull request by Saichethan**
### Description
Fixed broken links 

Fixes # [ISSUE]
#102

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: gsoc/2013.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
     <title>Systers Open Source Projects by Systers</title>
-
+
```
Comment (sara-02) [inline gsoc/2013.html]: @Saichethan these changes are unnecessary, mostly introduced by IDE's reformatting. Try not to push these commits by using `git add -p` to do a selective addition.

Comment (Saichethan) [reply gsoc/2013.html]: Actually, I Made changes to meta description in this branch but to make them independent I removed them later.

Comment (AnkitKhatri55555) [reply gsoc/2013.html]: @sara-02 , I think similar issue was opened, #80

Comment (AnkitKhatri55555) [reply gsoc/2013.html]: ya @sara-02 ,  RGSoC and Outreachy  names have changed

Comment (Saichethan) [reply gsoc/2013.html]: Should I change something else?

Comment (sara-02) [reply gsoc/2013.html]: @Saichethan No we are good with this as of now.

Comment (Saichethan) [reply gsoc/2013.html]: @sara-02 , I think I changed let me know if there is anything I should do.

**Comment by Tharangi** [comment]
This is fixed in the current version of http://systers.io/outreachy/2017.html. Also, RGSoC is no more there. 
image
@divyanshu-rawat , @Janiceilene: Shall we close this PR and the issue #102 as well?

**Comment by Tharangi** [comment]
Closing as this is fixed now.

**Pull request closed without merging**